### ISSUE TITLE
Keyboard help dialog: change vs choose

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#95]
In the third step of "Change Solute" section, should be "choose solute" instead of "change solute"

**Comment by terracoda** [comment]
@twant, this is probably on your radar, but just in case, you need to update the a11y content for the dialog as well as the visual content.
<img width="562" alt="molarity-dialog" src="https://user-images.githubusercontent.com/1902116/58081136-ce7ae100-7bb4-11e9-8489-15c0f0a24eec.png">

**Comment by terracoda** [comment]
@twant, also, there is no way you would have know this (so apologies), but the square brackets in the design doc are meant to communicate that an icon visually goes in that spot in the visual dialog, but the brackets are not meant to be included in the dialog content.

Please remove all the square brackets in the a11y content for the dialog. The brackets are only appearing in the the list items under the "Change Solute in 3 Steps" heading.

Again, apologies for not making our communication in the design doc more clear.

**Comment by terracoda** [comment]
I am also going to create a related issue about changing about adding a numbered list for the Steps to a Solute.

**Comment by terracoda** [comment]
@twant, I made a commit that referenced #96 and #95, but changes are just to #95.
Still need to update the heading, though. Please do that.

**Comment by terracoda** [comment]
The only thing left for this issue is the visual change from "Change" to "Choose" noted in the initial comment. I removed the square brackets and edited the content to read "Choose".

**Comment by twant** [comment]
@terracoda just to confirm: The heading for the section should read "Choose Solute in 3 Steps," not "Change Solute in 3 Steps?"

**Comment by twant** [comment]
Nevermind, just read the other issue. Will implement the new heading now (Steps to Change Solute).

**Comment by twant** [comment]
Closing so we can consolidate to just one issue tracking this (#96).
